pocs/camera/canon_gphoto2.py

```diff
@@ -139,7 +139,11 @@ def take_observation(self, observation, headers=None, filename=None, **kwargs):
             'start_time': start_time,
         }
         metadata.update(headers)
+
+        # This can be different than for the headers because passed as param
```

Comment:
Not quite sure how to interpret that comment.

Reply:
Yeah, not me best comment. :) Will try to clarify.